lib/primer/forms/primer_text_field.ts

```diff
@@ -57,6 +57,7 @@ export class PrimerTextFieldElement extends HTMLElement {
   clearContents() {
     this.inputElement.value = ''
     this.inputElement.focus()
+    this.inputElement.dispatchEvent(new Event('input', { bubbles: true, cancelable: true}))
```

Comment:
```suggestion
    this.inputElement.dispatchEvent(new Event('input', {bubbles: true, cancelable: true}))
```

Reply:
Is this event actually cancelable? If so, we need to add logic to handle the case when it's canceled, eg:

```typescript
if (!this.inputElement.dispatchEvent(...)) return
```

Reply:
You're right, this should not be cancelable. Thanks!